plugin/pkg/scheduler/factory/factory.go

```diff
@@ -338,6 +338,11 @@ func (f *ConfigFactory) CreateFromConfig(policy schedulerapi.Policy) (*scheduler
 			}
 		}
 	}
+	// issue: https://github.com/kubernetes/kubernetes/issues/43845
+	// for backward compatibility if user set the value in policy file, then we use it
```

Comment:
@wanghaoran1988 Backward compatibility applies if the user does not set it in the policy file and keep using the old way of providing the value, i.e., as a command line argument. The scenario that you described is the new way of providing the value. Nothing is wrong with your logic, just the comment is misleading.